.github/workflows/ci.yml

```diff
@@ -14,16 +14,16 @@ concurrency:
 
 jobs:
   static:
-    uses: dvsa/.github/.github/workflows/php-library-static.yml@v3.2.3
+    uses: dvsa/.github/.github/workflows/php-static.yml@v4.0.1
+    with:
+      php-version: "8.1"
 
   test:
-    uses: dvsa/.github/.github/workflows/php-library-tests.yml@v3.2.3
+    uses: dvsa/.github/.github/workflows/php-tests.yml@v3.2.3
     with:
-      php-versions: "[\"8.0\", \"8.1\"]"
+      php-versions: "[\"8.1\"]"
 
   security:
-    uses: dvsa/.github/.github/workflows/php-library-security.yml@v3.2.3
-    with:
-     php-versions: "[\"8.0\", \"8.1\"]"
-    secrets:
+    uses: dvsa/.github/.github/workflows/php-security.yml@v3.2.3
```

Comment:
Can we use version @4.1.0 like in this PR: [mot-doctrine-module](https://github.com/dvsa/mot-doctrine-module/pull/10)?

Reply:
I've bumped them all to 4.1.0 using php version 8.1 and composer version 2.2.24. My local is running composer 2.7.x, so I figured 2.2.24 would be fine and it seems to be ok.